samples/daal/cpp/kdb/README.md

```diff
@@ -4,6 +4,8 @@ KDB\* samples for the Intel(R) Data Analytics Acceleration Library (Intel(R) DAA
 
 Unzip the archive with Intel(R) DAAL samples to your working directory (`<sample_dir>`).
 
+**Note:** The Intel® oneAPI Data Analytics Library (oneDAL)  is able to interface with kdb+. **Intel does not provide kdb+ or any license to Kdb+ as part of oneDAL.**  kdb+ is available for license from Kx Systems, Inc. under a commercial license (which may include fees) as well as a non-commercial license. You are solely responsible for procuring any license for kdb+ based on your use case.
```

Comment:
```suggestion
**Note:** The Intel® oneAPI Data Analytics Library (oneDAL) is able to interface with kdb+. **Intel does not provide kdb+ or any license to kdb+ as part of oneDAL.** kdb+ is available for license from Kx Systems, Inc. under a commercial license (which may include fees) as well as a non-commercial license. You are solely responsible for procuring any license for kdb+ based on your use case.
```

I'd also add `---` on the line above that note to produce a horizontal line that separates the note from the other text.

Reply:
It should be fixed now.